exercises/identify-conflicts.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Identify Conflicts
+tags:
+  - exercise
+  - converge
+---
+Throughout Diverge there should have been many conflicting ideas. A conflict is a place where there are two or more different approaches to solving the same problem. Conflicting approaches are helpful because they illuminate possible choices for your product.
```

Comment:
```suggestion

---
Throughout Diverge there should have been many conflicting ideas. A conflict is a place where there are two or more different approaches to solving the same problem. Conflicting approaches are helpful because they illuminate possible choices for your product.
```